lib/trailblazer/operation/dry_schema.rb

```diff
@@ -0,0 +1,32 @@
+module Trailblazer
+  class Operation
```

Comment:
Why do we need this file? Those features are already implemented in Reform 2.3.0.rc :thinking:

Reply:
Oh, ok, the whole point of the `Schema` is to not couple this to Reform. What we actually want to do is as follows:

1. Introduce a generic `Errors` object that is passed through operations (could be a separate gem).
2. Allow that object to deal with AMV and Dry::V results
3. Change `Contract::` macros to use that object.

I've been chatting about this with @fran-worley and I think it's time to tackle it. Also @patcoll